sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/coverage/CoberturaParser.java

```diff
@@ -37,6 +37,14 @@
  * {@inheritDoc}
  */
 public class CoberturaParser implements CoverageParser {
```

Comment:
@abbeyj looks like a specific solution for CoberturaParser. Could you move this ```baseDir``` to ```CoverageParser``` to make it more generic please.